spec/unit/knife_spec.rb

```diff
@@ -156,8 +156,8 @@ class AwesomeCheffsCommand < Chef::ChefFS::Knife
                     "Accept-Encoding"=>"gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
                     'X-Chef-Version' => Chef::VERSION,
                     "Host"=>"api.opscode.piab",
-                    "X-REMOTE-REQUEST-ID"=>request_id,
-                    'X-Ops-Server-API-Version' => Chef::HTTP::Authenticator::DEFAULT_SERVER_API_VERSION}}
```

Comment:
It's not an auth'd request (rest is an instance of NoAuthHTTP (above)), so it's not hitting the Authenticator at all. We only use the noauth_rest stuff for get requests to supermarket.